.github/workflows/build-cpu-windows.yml

```diff
@@ -0,0 +1,27 @@
+name: build-windows
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        #os: [windows-latest, windows-2016]
+        os: [windows-latest]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Configure
+      run: |
+        mkdir build
+        cd build
+        cmake -DBUILD_SUPERBUILD=ON -DBUILD_SHARED_LIBS=OFF ..
```

Comment:
Why "-D DBUILD_SHARED_LIBS=OFF"? By default it is ON (master)

Reply:
To avoid having DLL files in the system path. This is just under Windows.